src/main/fulcro/client/primitives.cljc

```diff
@@ -3211,22 +3177,13 @@
                 :clj  clojure.lang.Atom) x))
 
 (defn integrate-ident!
-  "Integrate an ident into any number of places in the app state. This function is safe to use within mutation
-  implementations as a general helper function.
-
-  The named parameters can be specified any number of times. They are:
-
-  - append:  A vector (path) to a list in your app state where this new object's ident should be appended. Will not append
-  the ident if that ident is already in the list.
-  - prepend: A vector (path) to a list in your app state where this new object's ident should be prepended. Will not append
-  the ident if that ident is already in the list.
-  - replace: A vector (path) to a specific location in app-state where this object's ident should be placed. Can target a to-one or to-many.
-   If the target is a vector element then that element must already exist in the vector.
-  "
+  "DEPRECATED: Use fulcro.client.mutations/integrate-ident* in your mutations instead."
```

Comment:
Same comment: "integrate-ident! is deprecated and will be removed in a future version. Please use..."